examples/experiments/tracker_dealer_doublesided.py

```diff
@@ -69,25 +69,24 @@ def callback(event_type, value=None, timestamp=None):
                    change_rule, x_min, x_max) for sv in start_value]
 
 # initialize TrackerDealer object
-tr = TrackerDealer(tr_ud, max_lag, rng_dealer)
+td = TrackerDealer(callback, tr_ud, max_lag, rng_dealer)
 
 # Initialize human state
 rng_human = np.random.RandomState(1)  # random seed for modeled subject
 
-while not tr.stopped:
+for _, level in td:
     # Get information of which trial type is next and what the level is at
     # that time from TrackerDealer
-    _, level = tr.get_trial()
-    tr.respond(rng_human.rand() < sigmoid(level - true_thresh,
+    td.respond(rng_human.rand() < sigmoid(level - true_thresh,
                                           lower=chance, slope=slope))
 
 ##############################################################################
 # Plotting the Results
 # ---------------------------
 axes = plt.subplots(2, 1)[1]
 for i in [0, 1]:
-    fig, ax, lines = tr[i].plot(ax=axes[i])
-    lines += tr[i].plot_thresh(4, ax=ax)
+    fig, ax, lines = td.trackers.ravel()[i].plot(ax=axes[i])
+    lines += td.trackers.ravel()[i].plot_thresh(4, ax=ax)

```

Comment:
BTW all of the `set_label` calls below should probably live in `tracker.plot()`. When doing `ax.plot(...)` you can do `ax.plot(..., label=...)`.